### PR TITLE
Dennis css

### PR DESCRIPTION
**Pull request by kurantoB**
fixes the text and extra side-scrolling from the padding
remove redundant classname

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -10,7 +10,8 @@ import './layouts/Gallery.css';
 function App(props) {
     console.log(props);
     return (
-        <div className="Gallery">
+        // className="Gallery" probably unintentional, "Gallery" is already being used below
```
Comment (rain-and-cappuccino) [inline src/App.js:13]: oops, yea i was messing around with a lot of stuff and tried to get it to format

File: src/layouts/Gallery.css [hunk 2]
```diff
@@ -1,8 +1,14 @@
-.Gallery {
+/* This is to fix the size of the right section, .Gallery will take care of the flex and padding */
+.GalleryContainer {
     color: aliceblue;
-    width: 100vw;
+    width: calc(100% - 300px);
```
Comment (rain-and-cappuccino) [inline src/layouts/Gallery.css:4]: hmm, so this is based on the sidebar? so if i change the sidebar width, i would have to change this too?

File: src/layouts/Gallery.css [hunk 3]
```diff
@@ -1,8 +1,14 @@
...
     min-height: 100vh;
     background-color: #3d4e85; /* Black */
+    margin-left: 300px;
```
Comment (rain-and-cappuccino) [inline src/layouts/Gallery.css:7]: * and this

File: src/layouts/Gallery.css [hunk 4]
```diff
@@ -1,8 +1,14 @@
...
+}
+
+.Gallery {
     flex-direction: row;
-    padding: 40px 350px;
+    display: flex; /* otherwise, the default display is flow */
```
Comment (rain-and-cappuccino) [inline src/layouts/Gallery.css:12]: hmm, so this has the items auto-line-up nicely?

File: src/layouts/ImageCard.css [hunk 5]
```diff
@@ -10,13 +10,21 @@
...
     color: darkslateblue;
-    text-overflow: ellipsis;
-    overflow: hidden;
     display: inline-block;
     white-space: pre-line;
 }
 
+.ImageCard > p {
+    overflow: hidden;
+    text-overflow: ellipsis;
```
Comment (rain-and-cappuccino) [inline src/layouts/ImageCard.css:19]: I tried adding this cause google but i'm not sure if this is in the wrong place or i need a different property cause it's not really showing ellipsis even if i modify things :/

File: src/layouts/Gallery.css (hunk 2 above)
Comment (kurantoB) [reply src/layouts/Gallery.css:4]: Yeah.. css variables are a thing too but I'm too lazy to use them here

File: src/layouts/Gallery.css (hunk 4 above)
Comment (kurantoB) [reply src/layouts/Gallery.css:12]: flex-direction only takes effect if display is set to flex first

File: src/layouts/ImageCard.css (hunk 5 above)
Comment (kurantoB) [reply src/layouts/ImageCard.css:19]: Try removing white-space: pre-line from .ImageCard, adding it to .ImageCard > p as white-space: nowrap. Then you should see a single line with ellipses

File: src/layouts/Gallery.css (hunk 2 above)
Comment (rain-and-cappuccino) [reply src/layouts/Gallery.css:4]: will look into those! thanks! :D

File: src/layouts/Gallery.css (hunk 4 above)
Comment (rain-and-cappuccino) [reply src/layouts/Gallery.css:12]: ahh... that makes a lot of sense...

File: src/layouts/ImageCard.css (hunk 5 above)
Comment (rain-and-cappuccino) [reply src/layouts/ImageCard.css:19]: ooh it worked! i thought i tried some variation of this yesterday but maybe the sizing of something messed it up 🤔 
thanks!

Comment (kurantoB) [reply src/layouts/ImageCard.css:19]: Ayy

**Pull request merged**